docs/pages/project/contributing/contributing-docs.md

```diff
@@ -146,7 +146,13 @@ Meshery documentation is built using the following components:
   </code>
   </div></pre>
 
+- Install ruby-build as an rbenv plugin
 
+  <pre class="codeblock-pre"><div class="codeblock">
+  <code class="clipboardjs">git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build
```

Comment:
why do we need to clone. It is also possible to install first the rbenv package just normally .

Reply:
this is needed if you are following instructions from source

Reply:
@lekaf974 if this step can cause confusion add a note clarifying why and when this is needed (as pointed out by @sudhanshutech)

Reply:
@Tharanishwaran can you add some description based on @sangramrath request

Reply:
@lekaf974 and @sangramrath thank you for your feedback. I've added a note to clarify when the ruby-build installation is necessary.